opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -1453,8 +1464,15 @@ File {} line {}.)", pattern, location.keyword, location.filename, location.linen
         this->snapshots.resize(reportStep + 1);
         auto& input_block = this->m_sched_deck[reportStep];
         std::unordered_map<std::string, double> wpimult_global_factor;
+        ScheduleLogger logger(ScheduleLogger::select_stream(true, true),
```

Comment:
This will log to the debug stream, right? Is this intentional?

Reply:
Yes, we could also log to OpmLog::info, I changed that